### PR TITLE
feature(setup) Setup project and add color palette

### PR DESCRIPTION
**Pull request by tomuhenry**
**Description of tasks to be completed**
- Go to www.materialpalette.com
- Add color palatte
- Add circleci badge
- Add code climate badge
- Setup the android project
- Add badges

**Relevant Pivotal Tracker stories**
[#159051218](https://www.pivotaltracker.com/story/show/159051218)
[#155738437](https://www.pivotaltracker.com/story/show/155738437)
[#155738456](https://www.pivotaltracker.com/story/show/155738456)
[#155738450](https://www.pivotaltracker.com/story/show/155738450)

Old Color Scheme
Screen Shot 2019-03-21 at 3 43 25 PM

Color Palatte
palette

New Color Scheme
Screen Shot 2019-03-21 at 3 45 38 PM

### CODE REVIEW

**Comment by tomuhenry** [comment]
Hey @damafeez, the badges are pointing to the **develop** branch and that is why the build is failing. But the might will be able to solve that.

File: .idea/libraries/Gradle__android_arch_core_common_1_1_1_jar.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<component name="libraryTable">
```
Comment (damafeez) [inline .idea/libraries/Gradle__android_arch_core_common_1_1_1_jar.xml]: You don't interact directly with files in the `/.idea/libraries` path, you ignore them.
You can add `/.idea/libraries` to your `.gitignore`

**Review by damafeez — CHANGES_REQUESTED** [review]
Good job @tomuhenry. This PR contains some files you don't directly interact with, however. You can add these paths to your `.gitignore`
```
*.iml
.gradle
/local.properties
/.idea/caches/build_file_checksums.ser
/.idea/libraries
/.idea/modules.xml
/.idea/workspace.xml
.DS_Store
/build
/captures
.externalNativeBuild
```

File: .idea/libraries/Gradle__android_arch_core_common_1_1_1_jar.xml (hunk 1 above)
Comment (tomuhenry) [reply .idea/libraries/Gradle__android_arch_core_common_1_1_1_jar.xml]: Added the files to the `.gitignore file`

**Review by damafeez — APPROVED** [review]
LGTM, I'm merging this

**Pull request merged**